src/account/UpgradeableModularAccount.sol

```diff
@@ -109,21 +109,21 @@ contract UpgradeableModularAccount is
 
     /// @notice Initializes the account with a set of plugins
     /// @param plugins The plugins to install
-    /// @param manifestHashes The manifest hashes of the plugins to install
+    /// @param manifests The manifests of the plugins to install
     /// @param pluginInstallDatas The plugin install datas of the plugins to install
     function initialize(
         address[] memory plugins,
-        bytes32[] memory manifestHashes,
+        PluginManifest[] calldata manifests,
         bytes[] memory pluginInstallDatas
     ) external initializer {
         uint256 length = plugins.length;
 
-        if (length != manifestHashes.length || length != pluginInstallDatas.length) {
+        if (length != manifests.length || length != pluginInstallDatas.length) {
```

Comment:
I guess we could pack pluginInstallDatas into the same struct and skip this check altogether since we're no longer just passing an array of raw hashes

Reply:
Yeah, this initialization sequence is also not used anymore, since there's a validation-specific initialization now. Possibly worth removing.

Reply:
Sounds good to me! Are there any potential uses where this would be necessary (in an atomic context, deploy & init w/ plugin) that you can think of?

Anyway this could be done using a UserOp if push comes to shove, yeah? But besides that it wouldn't be possible to do so atomically without a custom deployer that then executes stuff (correct me if I'm wrong).

Reply:
Some factories may wish to have some more heavyweight initialization, like installing multiple plugins. Though currently, as soon as one initializer runs, the other is disabled, so this doesn't address that. A clumsy workaround is to add the factory as an owner mid-deploy, then give up ownership by the end of the deploy.

Either way, it's for a different workflow than what this PR is working on. I'll remove this initializer in #104, because that's the PR where validation is removed as an installable component via the manifest.